app/models/registration_payment.rb

```diff
@@ -23,7 +23,7 @@ def amount_available_for_refund
   end
 
   private def should_auto_accept?
-    auto_accept_in_current_env? && auto_accept_registrations && Registration::LIVE_AUTO_ACCEPT_ENABLED
+    auto_accept_in_current_env? && auto_accept_registrations? && Registration::LIVE_AUTO_ACCEPT_ENABLED
```

Comment:
You're still calling `auto_accept_in_current_env?` here, despite deleting the `delegate` above. How does this work?

Reply:
Fixed - I presume this was not reported by the test suite because we're expecting this hook not to run